package.json

```diff
@@ -29,10 +29,11 @@
     "lint": "eslint .",
     "format": "eslint . --fix",
     "flow": "flow src",
-    "test": "cross-env NODE_ENV=test jest",
+    "test": "cross-@descript/draft-jsenv NODE_ENV=test jest",
```

Comment:
was this intentional, or a copy paste artifact?

Reply:
bug - will revert!